packages/node/src/handlers/test-api.ts

```diff
@@ -36,8 +44,8 @@ export async function testApi(
     clientAddress: '',
     designatedWallet: '',
     chainId: '',
-    endpointId: triggerRequest.endpointId,
-    endpointName,
+    endpointId,
+    endpointName: triggerRequest.endpointName,
```

Comment:
Github doesn't allow me to comment on non changed lines, but can you please fix a typo on line 61?

`An unkNown error occurred`